cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityUserOperator.java

```diff
@@ -198,6 +230,8 @@ public static EntityUserOperator fromCrd(Kafka kafkaAssembly) {
         varList.add(buildEnvVar(ENV_VAR_CLIENTS_CA_KEY_SECRET_NAME, KafkaCluster.clientsCaKeySecretName(cluster)));
         varList.add(buildEnvVar(ENV_VAR_CLIENTS_CA_CERT_SECRET_NAME, KafkaCluster.clientsCaCertSecretName(cluster)));
         varList.add(buildEnvVar(ENV_VAR_STRIMZI_GC_LOG_ENABLED, String.valueOf(gcLoggingEnabled)));
+        varList.add(buildEnvVar(ENV_VAR_CLIENTS_CA_VALIDITY, Long.toString(clientsCaValidityDays)));
+        varList.add(buildEnvVar(ENV_VAR_CLIENTS_CA_RENEWAL, Long.toString(clientsCaRenewalDays)));
```

Comment:
Why are you using `Long` here but then `Integer` in the test. I guess `Integer` is ok for both.